python/pyarrow/compute.py

```diff
@@ -591,6 +591,38 @@ def take(data, indices, *, boundscheck=True, memory_pool=None):
     return call_function('take', [data, indices], options, memory_pool)
 
 
+def drop_null(data, memory_pool=None):
```

Comment:
I'll remove this as the proper definition is auto-generated from the C++ metadata.